manifests/init.pp

```diff
@@ -23,6 +23,7 @@
   $repo_proxy_password                                 = $nodejs::params::repo_proxy_password,
   $repo_proxy_username                                 = $nodejs::params::repo_proxy_username,
   $repo_url_suffix                                     = $nodejs::params::repo_url_suffix,
+  $repo_release                                        = $nodejs::params::repo_release,
```

Comment:
Please add an entry to the README for this

Reply:
Added README entry
https://github.com/voxpupuli/puppet-nodejs/pull/342/commits/40f96bb2035b578ec1602581c822c170e7028e6a

Reply:
Can you add a datatype for the new param?